packages/flutter/lib/src/rendering/sliver_persistent_header.dart

```diff
@@ -348,11 +348,7 @@ abstract class RenderSliverScrollingPersistentHeader extends RenderSliverPersist
     }
     final double maxExtent = this.maxExtent;
     final double paintExtent = maxExtent - constraints.scrollOffset;
-    final double cacheExtent = calculateCacheOffset(
-      constraints,
-      from: 0.0,
-      to: maxExtent,
-    );
+    final double cacheExtent = calculateCacheOffset(constraints, from: 0.0, to: maxExtent);
```

Comment:
Could you please leave this as-is to keep the line under 80 chars?

https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#prefer-a-maximum-line-length-of-80-characters

Reply:
I changed it. Alouthgh  In my opinion, it is most applicable to the second description:

> but prefer going over if breaking the line would make it less readable, or if it would make the line less consistent with other nearby lines.

```dart
abstract class RenderSliverPersistentHeader extends RenderSliver with RenderObjectWithChildMixin<RenderBox>, RenderSliverHelpers {
  //                                                                                                                             ^ 130 chars 
  // ......

  @override
  void debugFillProperties(DiagnosticPropertiesBuilder properties) {
    // ......
    properties.add(DoubleProperty.lazy('child position', () => childMainAxisPosition(child!)));
    //                                                                                        ^ 95 chars 
  }
}

abstract class RenderSliverScrollingPersistentHeader extends RenderSliverPersistentHeader {
  //                                                                                      ^ 91 chars 
  RenderSliverScrollingPersistentHeader({
    super.child,
    super.stretchConfiguration,
  });

  double? _childPosition;

  @protected
  double updateGeometry() {
    double stretchOffset = 0.0;
    if (stretchConfiguration != null) {
      stretchOffset += constraints.overlap.abs();
    }
    final double maxExtent = this.maxExtent;
    final double paintExtent = maxExtent - constraints.scrollOffset;
    final double cacheExtent = calculateCacheOffset(constraints, from: 0.0, to: maxExtent);
    //                                                                                    ^ 91 chars

    geometry = SliverGeometry(
      cacheExtent: cacheExtent,
      scrollExtent: maxExtent,
      paintOrigin: math.min(constraints.overlap, 0.0),
      paintExtent: clampDouble(paintExtent, 0.0, constraints.remainingPaintExtent),
      //                                                                          ^ 83 chars
      maxPaintExtent: maxExtent + stretchOffset,
      hasVisualOverflow: true, // Conservatively say we do have overflow to avoid complexity.
      //                                                                                    ^ 93 chars
    );
    return stretchOffset > 0 ? 0.0 : math.min(0.0, paintExtent - childExtent);
  }

  @override
  void performLayout() {
    layoutChild(constraints.scrollOffset, maxExtent);
    _childPosition = updateGeometry();
  }

  @override
  double childMainAxisPosition(RenderBox child) {
    assert(child == this.child);
    assert(_childPosition != null);
    return _childPosition!;
  }
}

abstract class RenderSliverPinnedPersistentHeader extends RenderSliverPersistentHeader {
  //                                                                                   ^ 88 chars 
  // ......
}
```